src/pages/index.js

```diff
@@ -58,41 +148,63 @@ addCardButton.addEventListener("click", () => {
   newCardPopup.open();
 });
 
-// Initialize an instance
-newCardPopup.setEventListeners();
-
-// UserInfo popup
-// Comment to the reviewer - I changed here the logics to use
-// the PopupWithFormClass for Profile Data change
-// and removed the extension of the popup class to the UserInfo class
-
+// UserInfo Popup
 const userPopupForm = new PopupWithForm({
   popupSelector: "#profileChange",
   handleFormSubmit: (userInput) => {
-    const newUserInput = userInput;
-    userInfo.setUserInfo({
-      name: newUserInput.name,
-      description: newUserInput.description,
-    });
-    userPopupForm.close();
+    userPopupForm.renderLoading(true);
+    api
+      .editProfileData(userInput)
+      .then((data) => {
+        userInfo.setUserInfo({
+          name: data.name,
+          description: data.about,
+        });
+        userPopupForm.close();
+      })
+      .catch((err) => console.error(err))
+      .finally(() => userPopupForm.renderLoading(false));
   },
 });
 
+// PROFILE INFO CHANGES
+
+// User Info and Avatar Change
 const userInfo = new UserInfo({
   nameSelector: ".profile__title",
   descriptionSelector: ".profile__subtitle",
+  avatarSelector: ".profile__avatar",
+});
+
```

Comment:
* `popup.js` should be in **UpperCase**: `Popup.js`


If the file name doesn't change on `github`, then it's a problem of `git cache`. Please, clear it like this:

```bash

git rm -r --cached .
git add .
git commit -am 'git cache cleared'
git push

```

Reply:
I cleared the cache in Git index, it indeed was in the lower letter there, but in my local repository already with the capital letter.

Regarding likes and hearts ( that after reload they dissapear ). I don't know, what is the server logics there, because I send 2 requests - PUT and DELETE on the like and dislike and send this data to the server. I am not sure I can do anything with that on the client.

Reply:
> Regarding likes and hearts ( that after reload they dissapear ). I don't know, what is the server logics there, because I send 2 requests - PUT and DELETE on the like and dislike and send this data to the server. I am not sure I can do anything with that on the clien

When you get the initial cards, they have `isLiked` inside each object. This is what you need to make the hearts black (liked) after reloading